hosted_control_planes/hcp-managing.adoc

```diff
@@ -20,4 +20,5 @@ include::modules/hosted-control-planes-pause-reconciliation.adoc[leveloffset=+1]
 //configuring metrics sets
 //automated machine management
 include::modules/delete-hosted-cluster.adoc[leveloffset=+1]
+include::modules/hosted-control-planes-troubleshooting.adoc[leveloffset=+1]
```

Comment:
Just wondering if it would make sense to have this come first and then have deleting a hosted cluster be last